packages/ui/primitives/document-flow/add-fields.tsx

```diff
@@ -596,9 +615,16 @@ export const AddFieldsFormPartial = ({
             previousStep();
             remove();
           }}
-          onGoNextClick={() => void onFormSubmit()}
+          onGoNextClick={handleGoNextClick}
         />
       </DocumentFlowFormContainerFooter>
+
+      {!everySignerHasSignature && (
```

Comment:
Any reason to conditionally render here? I get why it's there but it seems easier to just let it always render since we toggle visibility anyways.